test/static-code

```diff
@@ -10,14 +10,18 @@ set -eu
 
 # note: `set -e` is not active during the tests.
 
+exclude_external() {
+    grep -v '^test/common/fmf_metadata'
```

Comment:
I was kinda wondering why it worked at all since it contained an outright error.  I considered dropping it as well to see if it even broke anything.  Happy to see this go.

Reply:
Dropped this again, thanks.